api/src/components/crosslink/controller.ts

```diff
@@ -152,10 +157,37 @@ export const getPublicationCrosslinks = async (
             return response.json(404, { message: 'Publication not found.' });
         }
 
-        const crosslinks = await crosslinkService.getPublicationCrosslinks(
-            event.pathParameters.publicationId,
-            event.queryStringParameters.order || undefined
-        );
+        const ownLinks = event.queryStringParameters.own === 'true';
+
+        if (ownLinks && !event.user) {
+            return response.json(400, {
+                message: 'Cannot filter to your own items when the request is not authenticated.'
+            });
+        }
+
+        if (
+            event.queryStringParameters.order === 'mix' &&
+            (event.queryStringParameters.limit || event.queryStringParameters.offset)
+        ) {
+            return response.json(400, {
+                message: 'Pagination cannot be used with "mix" sort order.'
+            });
+        }
+
+        const search = event.queryStringParameters?.search
```

Comment:
Curious why this one needs the Optional Chaining but all the others above don't

Reply:
Not sure why it was there, it's not needed!